railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_5_2.rb.tt

```diff
@@ -0,0 +1,3 @@
+# Use AES 256 GCM authenticated encryption for encrypted cookies.
+# Existing cookies will be converted on read then written with the new scheme.
+Rails.application.config.use_authenticated_cookie_encryption = false
```

Comment:
We need to copy over the intro from the `_5_1.rb` file and nestle the config to be under `config.action_dispatch`.